clothing_app/clothing_app/pipelines.py

```diff
@@ -11,6 +11,21 @@
 
 class ClothingAppPipeline:
     def process_item(self, item, spider):
+        item = self.typecast_values(item)
+        item = self.fill_essential_values(item)
+        return item
+
+    def typecast_values(self, item):
+        if item.get("old_price_text"):
```

Comment:
The price is usually accompanied by currency symbols. e.g;
`$4.99` OR `15 GPB` OR `PKR: 2,000`. We cannot directly typecast the values.